README.md

```diff
@@ -1,70 +1,197 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages capitalized to make your pull request professional enough 👍 😊

Reply:
Thank you for your advice. I'm not sure if it's possible to change old commit messages or if changing them is an acceptable practice.
But I will keep it in mind for future projects 😊.

Many thanks.